index.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-sparse-arrays */
+function calculate(arr) {
+  let isOperand = ['+', '-', '*', '/']
+  let invalidInputs = ['', , undefined, null]
```

Comment:
The `, ,` ends up being undefined so you don't need it here since you also have undefined.